plugins/@grouparoo/iterable/src/lib/test.ts

```diff
@@ -0,0 +1,9 @@
+import { TestPluginMethod } from "@grouparoo/core";
+import { connect } from "./connect";
+
+export const test: TestPluginMethod = async ({ appOptions }) => {
+  const client = await connect(appOptions);
+  const success = await client.lists.get();
+  const message = success ? 'Connected to Iterable api.' : null;
```

Comment:
What is success here? An array of lists?
We try to have a message something like.

```
  const response = await client.lists.get();
  const success = !!response.lists;
  const message = success ? 'Found ${response.lists.length} lists' : `Error getting lists`;
```